packages/wallets/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCard.tsx

```diff
@@ -49,7 +61,11 @@ const JurisdictionCard: React.FC<TJurisdictionCardProps> = ({ isSelected, jurisd
                             key={description?.text}
                             onClick={e => {
```

Comment:
the same about onClickHandler, but again you can ignore it :)

Reply:
Updated.